### PR TITLE
go 1.7.1

### PR DESCRIPTION
**Pull request by cblecker**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

Modified description to pass `brew audit --new-formula Formula/go.rb`. Used title from https://golang.org/

### CODE REVIEW

**Comment by DomT4** [comment]
@BrewTestBot test this please

File: Formula/go.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
 class Go < Formula
-  desc "Go programming environment"
+  desc "The Go Programming Language"
```
Comment (zmwangx) [inline Formula/go.rb]: It's preferable to use Sentence case rather than Title Case for `desc`.

**Comment by cblecker** [comment]
@zmwangx Updated `desc` to use sentence case.

**Comment by cblecker** [comment]
All looks ✅ here

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/ce48dc5b18401a6d84331c10a2a050b673b4734e. Thank you for another contribution to Homebrew @cblecker; we appreciate it! :smiley_cat:

**Pull request closed without merging**